libs/blocks/aside/aside.js

```diff
@@ -226,6 +233,7 @@ export default function init(el) {
   decorateStaticLinks(el);
   formatPromoButton(el);
   decorateTextOverrides(el);
+  decorateA11yForLinks(el.querySelector('.action-area'), el.querySelector('h3')?.textContent?.trim());
```

Comment:
Should we be looking at any h tag rather than a specific h3? 
In the after page for aside, some of the examples using an h2 don't get this aria-label